src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+
+    add(a) {
+      calc.result += a;
+
+      return calc;
+    },
+
+    subtract(a) {
+      calc.result -= a;
+
+      return calc;
+    },
+
+    multiply(a) {
+      calc.result *= a;
+
+      return calc;
+    },
+
+    divide(a) {
+      calc.result /= a;
+
+      return calc;
+    },
```

Comment:
You need to add a check for 0, because by the rules of mathematics, you cannot divide by 0